src/search/airportsearch.cpp

```diff
@@ -744,14 +750,166 @@ void AirportSearch::randomFlightplanClicked()
 {
   Ui::MainWindow *ui = NavApp::getMainUi();
 
+  int distanceMin = ui->spinBoxAirportFlightplanMinSearch->value();
+  int distanceMax = ui->spinBoxAirportFlightplanMaxSearch->value();
+
   // Log minimum and maximum distance from UI
-  qDebug() << Q_FUNC_INFO << "min" << ui->spinBoxAirportFlightplanMinSearch->value()
-           << "max" << ui->spinBoxAirportFlightplanMaxSearch->value();
+  qDebug() << Q_FUNC_INFO << "random flight, distance min: " << distanceMin
+           << ", random flight, distance max: " << distanceMax;
+
+  if(distanceMin < distanceMax) {
+    // TODO:
+    // non-blocking notification: the closer the distance (max - min) is to 0, the longer the random selection might take
```

Comment:
I'd avoid this by keeping max-min over 100 NM or whatever makes sense. See also last comment.

Reply:
i like to search for as low as 30 thus i put it at 10 (which is the step size)

Reply:
Ok. No prob.